modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -75,6 +75,36 @@ definition {
 		return ${curl};
 	}
 
+	macro _getObjectEntryById {
```

Comment:
I think we can rename this to from _getObjectEntryId to getObjectEntryById since it is called in some of the testcase files. And we can remove the following duplicategetObjectEntryById macro.